Objective-C/CBLReplicatorConfiguration.m

```diff
@@ -289,15 +287,14 @@ - (void) addCollections: (NSArray<CBLCollection*>*)collections
 }
 
 - (NSArray<CBLCollection*>*) collections {
-    _collections = _collectionConfigs.allKeys;
-    return _collections;
+    return _collectionConfigs.allKeys;
 }
 
 - (void) removeCollection:(CBLCollection *)collection {
     [_collectionConfigs removeObjectForKey: collection];
     
     // reset the database, when all collections are removed
-    if (_collections.count == 0) {
+    if (self.collections.count == 0) {
```

Comment:
It's simpler by checking _collectionConfigs.count.

```
if (_collectionConfigs.count == 0)
```

Reply:
ok